docs/reference/symbols.rst

```diff
@@ -7,59 +7,31 @@ Symbols
 Symbols are a model's decision variables, intermediate variables, constants, 
 and mathematical operations.
 
-.. _symbols_model_symbols:
+See :ref:`Symbols <intro_optimization_symbols>` for an introduction to
+working with symbols.
 
-Model Symbols
-=============
+.. currentmodule:: dwave.optimization.model
```

Comment:
```suggestion
Inherited Methods
============

.. currentmodule:: dwave.optimization.model
```

Currently the side navigation won't be helpful to new users:

![image](https://github.com/user-attachments/assets/00ab7a78-8dc2-4278-8922-c06edb094e34)

Reply:
I don't love `Inherited`. What about `Base Symbols`?

Reply:
Could that be "Base Classes" instead? "Base symbols" might be mistaken as a different flavor of symbols (talk about saturation of a word)

Reply:
```suggestion
Base Classes
============

.. currentmodule:: dwave.optimization.model
```